test/zdtm/static/socket_udp-corked.c

```diff
@@ -50,26 +53,66 @@ int main(int argc, char **argv)
 		pr_perror("Can't bind socket");
 		return 1;
 	}
-	ret = connect(sk1, (struct sockaddr *)&addr1, len);
+	
```

Comment:
Could you please remove the trailing white-space?

Reply:
> > > Judging from my page the 26 tests are all passed. Do I miss something?
> > 
> > 
> > Although this test appears to pass, I can see in the log that the checkpoint has failed.
> 
> The test is marked as `crfail`. So it is expected to fail. https://github.com/checkpoint-restore/criu/blob/criu-dev/test/zdtm/static/socket_udp-corked.desc

Sure, will do.